### DISCUSSION TITLE
Jupyter notebook

### DISCUSSION THREAD

**Discussion by ghost** [community/community#55480] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can I hide code within a jupyter notebook so it won't be public?

**Comment by Jordan99990** [comment]
You could create a css file and make a div with display property none and after you create your notebook just check the raw JSON for url and add at the add of the url '?raw=true' and open it in a new tab.
It will look something like this
%%html
<kink rel="stylesheet" type="text/css" href="https://raw.github....com/{username}/{repo}/{branch}/file.css>
replace the values inside {} with your Github username, name of your repository where the "file.css" is located and the branch name and lastly save the notebook.

**Comment by queenofcorgis** [comment]
@LeNorbyke @Jacques9 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
